packages/player-component/index.js

```diff
@@ -326,12 +326,15 @@ class Player extends Nanocomponent {
 
   renderPlayer () {
     const isAuthenticated = !!this.state.user.uid
+    const infoBarClasses = 'flex flex-row w-100 justify-center bb b--light-silver no-underline'
 
     return html`
       <div class="player-component flex flex-column h-100">
         ${!isAuthenticated
-          ? html`<a class="flex flex-row w-100 justify-center bb b--light-silver no-underline" href="/login" target="_blank" rel="noopener noreferer">Log in to listen to full song</a>`
-          : ''}
+          ? html`<a class=${infoBarClasses} href="/login" target="_blank" rel="noopener noreferer">Log in to listen to full song</a>`
+          : Number(this.state.user.credits) < 1
+            ? html`<a class=${infoBarClasses}>You have less than one credit remaining</a>`
+            : ''}
```

Comment:
As a side note: nested ternaries tend to be frowned upon due to the significantly increased reading complexity.  Might be nice to just figure this out outside of the return statement using some regular variables/if statements and then just render here. Just a thought though.

Reply:
Good call. Addressed here: https://github.com/resonatecoop/stream/pull/213/commits/3bbf330b7d87897a0a4a2185c21125e89a612adf